superset-frontend/src/explore/ExplorePage.tsx

```diff
@@ -33,11 +40,35 @@ import { fallbackExploreInitialData } from './fixtures';
 
 const loadErrorMessage = t('Failed to load chart data.');
 
-const fetchExploreData = (exploreUrlParams: URLSearchParams) =>
-  makeApi<{}, ExploreResponsePayload>({
-    method: 'GET',
-    endpoint: 'api/v1/explore/',
-  })(exploreUrlParams);
+interface ResultInterface {
+  result: {
+    form_data: QueryFormData;
+    slice: Slice;
+    dataset: Dataset;
+  };
+}
+
+const isResult = (rv: JsonObject): rv is ResultInterface =>
+  rv?.result?.form_data &&
+  rv?.result?.slice &&
+  rv?.result?.dataset &&
+  isDefined(rv?.result?.dataset?.id) &&
+  isDefined(rv?.result?.dataset?.uid);
```

Comment:
Are both `id` and `uid` required?

Reply:
The same logic as before. I will improve this type and code style in the future.

Reply:
removed uid checker